locales/en-US/app.ftl

```diff
@@ -402,10 +403,15 @@ MenuButtons--metaOverheadStatistics-mean = Mean
 MenuButtons--metaOverheadStatistics-max = Max
 MenuButtons--metaOverheadStatistics-min = Min
 MenuButtons--metaOverheadStatistics-statkeys-overhead = Overhead
+    .title = Time to sample all threads.
 MenuButtons--metaOverheadStatistics-statkeys-cleaning = Cleaning
+    .title = Time to discard expired data.
 MenuButtons--metaOverheadStatistics-statkeys-counter = Counter
+    .title = Time to gather all counters.
 MenuButtons--metaOverheadStatistics-statkeys-interval = Interval
+    .title = Observed interval between two samples.
 MenuButtons--metaOverheadStatistics-statkeys-lockings = Lockings
+    .title = Time to take the lock before sampling.
```

Comment:
Maybe _set_ instead of _take_, assuming I understand the context correctly?

Reply:
I believe it should be either `take` or `acquire`. This is about the mutexes in our back-end. We have a profiler mutex and threads try to acquire/take this with a lock. 

[Example cpp mutex lock](https://en.cppreference.com/w/cpp/thread/mutex/lock) (not the same, but idea is)

Thinking about it, I think acquire is a better alternative here.

Reply:
Maybe use `locks` instead of `the lock`, as it's an aggregated time (I think)?

Reply:
min and max values are single locking times, only the mean value is aggregated.